articles/flows/guides/device-auth/call-api-device-auth.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Call API Using Device Authorization Flow
```

Comment:
Should we modify the urls to `device-flow` instead of `device-auth`? Not strong opinion just, it is commonly known as device flow maybe it helps for SEO.

Reply:
We usually make the URL close to the name in the spec, so if they decide to come out with another flow related to devices, we're able to distinguish easily between the two. I did change the keyword in the "topics" metadata, though.